examples/stubbing-spying__intercept/cypress/integration/glob-matching-intercept-url.spec.js

```diff
@@ -0,0 +1,13 @@
+describe('intercept', () => {
+    cy.intercept('/api/roles')
```

Comment:
Can we indent this file with 2 spaces? That seems to be the standard in this repo (but I may have missed it!)

Reply:
VSC was formatting with Prettier defaults. I fixed all broken eslint rules.